src/shared/configfile/configfile.h

```diff
@@ -20,7 +20,7 @@ Parses .gitconfig-style properties files.
 #ifdef WIN32
 
 #include <filesystem>
-#include "wslservice.h"
+#include "wslaservice.h"
```

Comment:
This change and other similar ones are probably an artifact of the previous merge. Those should be removed (it's preventing the CI from building)